include/rapidcheck/state/Command.h

```diff
@@ -27,6 +29,14 @@ class Command {
   /// properly.
   virtual void run(const Model &s0, Sut &sut) const;
 
+  /// Alterantive run function that allows rapidcheck to defer checking of the
+  /// system behaviour. This version must be used for parallel testing and can
+  /// be used for sequential testing.
+  ///
+  /// Use rapidcheck assertion macros in the returned function to check that the 
+  /// system behaves properly.
+  virtual std::function<void(const ModelT&)> run(Sut &sut) const;
```

Comment:
It doesn't matter but we should be consistent and use the typedef instead of the template parameter here. Thus, `ModelT` -> `Model`.